mycroft/skills/joke/__init__.py

```diff
@@ -34,7 +34,7 @@ def __init__(self):
         super(JokingSkill, self).__init__(name="JokingSkill")
 
     def initialize(self):
-        self.load_data_files(dirname(__file__))
+        self.load_vocab_files(join(dirname(__file__), 'vocab', 'en-us'))
```

Comment:
We have variables from MycroftSkill to the current language in use, please use that one:
`self.load_vocab_files(join(dirname(__file__), 'vocab', self.lang))`